crates/swc_ecma_transforms_base/src/resolver/mod.rs

```diff
@@ -133,6 +136,7 @@ struct Resolver<'a> {
     handle_types: bool,
     in_type: bool,
     in_ts_module: bool,
+    import_declarations: AHashMap<Id, bool>,
```

Comment:
Can you add a comment describing what information is stored as `bool`?

Reply:
actually I could just use hashset instead.